controlplane/pkg/apis/connectioncontext/connectioncontext.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 Red Hat, Inc.
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This contains the core Network Service Mesh API definitions for external
+// consumption via gRPC protobufs.
+
+syntax = "proto3";
+
+package connectioncontext;
+option go_package = "connectioncontext";
+
+message Route {
+    string dst_ip_addr = 1;             /* destination ip address + prefix in format <address>/<prefix> */
+    string gw_addr     = 2;             /* gateway address */
+    uint32 metric      = 3;             /* routing metric (weight) */
+}
+
+message ConnectionContext {
+    string src_ip_addr = 1;             /* source ip address + prefix in format <address>/<prefix> */
+    string dst_ip_addr = 2;             /* destination ip address + prefix in format <address>/<prefix> */
+
+    bool src_ip_reqiured = 3;           /* if specified src_ip is required from NSE. */
```

Comment:
type: src_ip_reqiured should be src_ip_required

Reply:
Thanks, Fixed.